cypress/support/pages/account/pageAccount.js

```diff
@@ -22,7 +25,11 @@ class Account {
 	}
 
 	enterCurrentPasswordOnUserSettingsPage() {
-		const currentPassword = Cypress.env("password");
+		const currentPassword =
+			Cypress.env("password") ||
+			Cypress.env("TEACHER_EXT_1_BRB_PASSWORD") ||
+			Cypress.env("TEACHER_EXT_1_DBC_PASSWORD") ||
+			Cypress.env("TEACHER_EXT_1_NBC_PASSWORD");
```

Comment:
password update is not required, so these condition can be removed now:
Cypress.env("TEACHER_EXT_1_BRB_PASSWORD") ||
Cypress.env("TEACHER_EXT_1_DBC_PASSWORD") ||
Cypress.env("TEACHER_EXT_1_NBC_PASSWORD");